routers/router.go

```diff
@@ -171,4 +171,10 @@ func initAPI() {
 
 	beego.Router("/.well-known/openid-configuration", &controllers.RootController{}, "GET:GetOidcDiscovery")
 	beego.Router("/.well-known/jwks", &controllers.RootController{}, "*:GetJwks")
+
+	beego.Router("/cas/:owner/:application/serviceValidate", &controllers.RootController{}, "GET:ServiceValidate")
+	beego.Router("/cas/:owner/:application/proxyValidate", &controllers.RootController{}, "GET:ServiceValidate")
+	beego.Router("/cas/:owner/:application/proxy", &controllers.RootController{}, "GET:Proxy")
+	beego.Router("/cas/:owner/:application/validate", &controllers.RootController{}, "GET:Validate")
```

Comment:
Validata -> CasValidate, similar for others

Reply:
@ComradeProgrammer with this bug, how does your demo site pass?

![image](https://user-images.githubusercontent.com/3787410/161299251-b68ec3a5-b4e8-4eb3-8e33-3806211def6f.png)

Reply:
@ComradeProgrammer change to:

- CasServiceAndProxyValidate
- CasProxy
- CasValidate